cmd/ef_tests/levm/report.rs

```diff
@@ -576,6 +580,79 @@ impl fmt::Display for ComparisonReport {
         for revm_updated_account_only in self.revm_updated_accounts_only.iter() {
             writeln!(f, "  {revm_updated_account_only:#x}:")?;
             writeln!(f, "    Was updated in REVM but not in LEVM")?;
+            let initial_account = self
+                .initial_accounts
+                .get(revm_updated_account_only)
+                .cloned()
+                .unwrap_or_default();
+            let updated_account_update = self
+                .revm_account_updates
+                .iter()
+                .find(|account_update| &account_update.address == revm_updated_account_only)
+                .unwrap();
+            let updated_account_storage = updated_account_update
+                .added_storage
+                .iter()
+                .map(|(key, value)| {
+                    let storage_slot = StorageSlot {
+                        original_value: *value,
```

Comment:
Shouldn't the original value be retrieved from the storage of the account pre-execution? Otherwise they are always going to have the same value, won't they?
If what I'm saying is correct I think we should find the corresponding storage slot in the `initial_account` and retrieve it's value.
We can also change this in the other for loop that iterates `levm_updated_accounts_only`

Reply:
You are right, we were also doing it incorrectly in the LEVM to REVM comparison, I updated both in [c712021](https://github.com/lambdaclass/ethrex/pull/1484/commits/c712021dda32d5b47d0409b8275fae984124d96a)